packages/components/organisms/f-checkout/src/services/mapper.js

```diff
@@ -0,0 +1,40 @@
+const mapCheckoutRequest = ({
```

Comment:
Should we be more specific in terms of what request to checkout this is about? Given we have various "checkout requests".